uber/site_sections/registration.py

```diff
@@ -132,6 +132,7 @@ def form(self, session, message='', return_to='', omit_badge='', check_in='', **
             'check_in':   check_in,
             'return_to':  return_to,
             'omit_badge': omit_badge,
+            'admin_can_change_status': session.admin_attendee().is_dept_head_of(c.REGDESK),
```

Comment:
Silly question: there's a Regdesk department and a Reg Managers department.  Are we sure we want this to be ``c.REGDESK`` and not ``c.REG_MANAGERS``?

Also, we should try to remember to clean this up after the event by e.g. making the department(s) configurable, since this will break Uber for events without a ``c.REGDESK`` department.

Reply:
Yea, I checked the regdesk dept heads in our production instance and they're the correct people!